capa/features/freeze.py

```diff
@@ -272,6 +272,14 @@ def main(argv=None):
         choices=(capa.main.BACKEND_VIV, capa.main.BACKEND_SMDA),
         default=capa.main.BACKEND_VIV,
     )
+    parser.add_argument(
+        "--signature",
+        action="append",
+        dest="signatures",
+        type=str,
+        default=[],
```

Comment:
would we want the default signatures here as well for consistency?

Reply:
or better yet use `install_common_args`?

Reply:
b89a29b